utilities/tools/Test-NamePrefixAvailability.ps1

```diff
@@ -34,6 +39,9 @@ function Test-NamePrefixAvailability {
         [Parameter(Mandatory = $true)]
         [string] $namePrefix,
 
+        [Parameter(Mandatory = $false)]
+        [string] $overwrittenServiceShort,
```

Comment:
Just to double-check  - what's the use case here if the service shot is always different (per file). Is it just for testing?

Reply:
Mainly for testing yes. Wanted to provide the ability if someone has a specific serviceshort in his mind to test this here too.